Android/app/src/main/java/io/github/project_travel_mate/utilities/UtilitiesFragment.java

```diff
@@ -60,6 +63,10 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         if (!hasAccelerometer || !hasMagneticSensor) {
             this.mHasMagneticSensor = false;
         }
+        String[] timeZones = TimeZone.getAvailableIDs();
```

Comment:
i dont think this is being used anywhere. You can remove this file from commit

Reply:
Used this to see what TimeZone.getAvailableIDs() returned so i can better understand what to modify. Forgot to remove when don. Now removed.